command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 01:50:01PM -0700, Mike Brown wrote:
> From my memory, the very reason the runtime spec did not have a
> client api was because we wanted to allow for other types of OCI
> products than just clients.

And we do allow that.  We just can't compliance-test a runtime to
certify it if it doesn't implement a specific API.  There's no
fundamental problem with specifying multiple APIs, but there is a
maintenance cost (the shim between the APIs is maintained by the OCI
complaince-testing team [1]).

> Why wouldn't we want to have interoperability with said other types
> of clients? What am I missing?

Interop on different APIs is great.  But we don't *need* that for
compliance testing, we just need one API.  After the first API, the
complaince testing team and runtime-spec maintainers can add
additional OCI-maintained APIs as they see fit.  And other
organizations can maintain their own alternative API specs as they see
fit (e.g. runC's current CLI docs).  But you still need agreement on
the APIs that the compliance-testers can understand.

[1]: https://github.com/opencontainers/runtime-spec/pull/513#issuecomment-231609551